packages/office-ui-fabric-react/src/components/ExtendedPicker/BaseExtendedPicker.tsx

```diff
@@ -159,7 +163,7 @@ export class BaseExtendedPicker<T, P extends IBaseExtendedPickerProps<T>>
     return onRenderSelectedItems({
       componentRef: this.selectedItemsList,
       selection: this.selection,
-      selectedItems: this.state.selectedItems ? this.state.selectedItems : undefined,
+      selectedItems: this.props.selectedItems ? this.props.selectedItems : undefined,
```

Comment:
There are still a lot of places where this.state.selectedItems is used. Do you need to change all of those as well?

Reply:
It doesn't need to be changed else where, since this is figuring out of the picker is being used a controlled component, it should be looking at the passed in props.selectedItems